packages/SwingSet/misc-tools/monitor-slog-block-time.py

```diff
@@ -98,7 +101,15 @@ def load_genesis_keys(fn):
 
```

Comment:
Suggested refactoring for `load_genesis_keys` above:
```python

def load_genesis_keys(fn):
    with open(fn) as f:
        keys = json.load(f)
    assert isinstance(keys, list)
    for validator in keys:
        assert isinstance(validator, list)
        (moniker, key_b64) = validator
        hexaddr = hashlib.sha256(base64.b64decode(key_b64)).hexdigest()[:40].upper()
        # hexaddr appears in `query block` | jq .block_header.proposer_address
        validators[hexaddr] = moniker
```

Reply:
I think the tuple unpacking will effectively perform the second assert (JSON doesn't distinguish between lists and tuples, but in python idiomatic terms `validator` would be a tuple because the items are of different types, so it would read a bit weird to have it assert list-ness). But I'll add the first assert.